src/config/index.ts

```
@@ -59,45 +59,160 @@ export async function loadCharacters(
+      case ModelProviderName.DEEPSEEK:
+          return (
+              character.settings?.secrets?.DEEPSEEK_API_KEY ||
+              settings.DEEPSEEK_API_KEY
+          );
+      case ModelProviderName.LIVEPEER:
+          return (
+              character.settings?.secrets?.LIVEPEER_GATEWAY_URL ||
+              settings.LIVEPEER_GATEWAY_URL
+          );
+      default:
+          const errorMessage = `Failed to get token - unsupported model provider: ${provider}`;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)